Stream workers index (#591)
Macrometacorp/docs: docs/cep/enrich-data/enrich-data-with-collection.md

## Create a Stream Worker

> Refer to the [Stream Workers](/cep/index.md) section for an explanation of how stream workers function.

Reviewer: ```suggestion
For more information about stream workers and how they work, refer [Stream Workers](index.md).
```

Reviewer: Generally we want to tell them what to expect first, then give them the instruction and link to click.

1. [Log in to your Macrometa account](https://auth-play.macrometa.io/).
1. Click **Stream Worker**.

> 1. In the Editor tab, you must define the stream worker. Copy and paste the following code block in the code editor on the Editor tab.

Reviewer: ```suggestion
1. In the Editor tab, define the stream worker. Copy and paste the following code block in the code editor on the Editor tab.
```

1. Select edge locations, and then click **Save**. The locations that you select represent where the data for this stream worker will live.
1. Click **Publish** to publish your stream worker. When you do this, it will begin to run the code as defined and will continue until you unpublish it.

> ## Stream Worker Explanation

Reviewer: Please leave the content here for now.

```sql
CREATE STREAM TransactionStream (userId long, transactionAmount double, location string);
```

#### Define the Table (Collection)

`CREATE TABLE` defines where the stream worker stores your data. In this case, it will be a [Document Store Collection](../../collections/documents/index.md) For more information about defining a TABLE in a stream worker, refer to [Table (Collection)](../query-guide/table-collection.md). For more information about collections in general, refer to [Collections](../../collections/index.md).

```sql
CREATE TABLE GLOBAL UserTable (userId long, firstName string, lastName string);
```

#### Define the Sink Stream

The sink stream is where the stream worker sends your data.

```sql
CREATE SINK EnrichedTransactionStream WITH (type='stream', stream='EnrichedTransactionStream', map.type='json') (userId long, userName string, transactionAmount double, location string);
```

> ### Data Enrichment Query

Reviewer: This content is only valid for this particular query, so it will likely remain here and unchanged permanently except for a link to the Stream Worker Queries section, once we have that.
